README.md

```diff
@@ -1 +1,138 @@
+- [x] HTML, CSS and JavaScript
+- [x] Linters: Lighthouse, Webhint, Stylelint and ESLint
+- [x] VS Code
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- [x] A web browser like Google Chrome
+- [x] A code editor like Visual Studio Code with Git and Node.js.
+
+### Setup
+
+Clone this repository to your desired folder using the link given below:
+
+use the git clone command to clone this repository or download ZIP
+
+### Install
+
+Install this project with:
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone https://github.com/Odongo006/math-magician-app
+```
+### Usage
+
+To run the project, execute the following command: 
+- Repository[@repo](https://github.com/Odongo006/math-magician-app)
+
+
+
+
+
+
+### Run tests
+
+To check for linters, use the files provided by [Microverse](https://github.com/microverseinc/linters-config). A GitHub action is also set to run during pull request.
+```
+$ npm install
+```
+
+### Key Features
+- Interactive games: The website offers a variety of games that allow children to practice their math skills in a fun and interactive way.
+- Instant feedback: The website provides instant feedback on whether the child's answer is correct or incorrect, allowing them to quickly learn from their mistakes.
+- Variety of math operations: The games cover addition, subtraction, multiplication, and division, allowing children to practice all of the basic math operations.
+
+### Future Features
+
+- Different skill levels: The games are categorized according to skill level, making it easy for children to find games that are appropriate for their level.
+- Free access: The Math Magician project is completely free to access and use, making it accessible to anyone with an internet connection.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Henry**
+
+- GitHub: [@Odongo006](https://github.com/Odongo006)
+- Twitter: [@HenryOdongo007](https://twitter.com/HenryOdongo007)
+- LinkedIn: [@HenryOdongo](https://www.linkedin.com/in/henry-odongo-91b830182/)
+- Website: [@Portfolio](https://odongo006.github.io/My-Portfolio-Project/)
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing "></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Odongo006/issues).
+
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give it a ⭐️
+
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- [Microverse](https://www.microverse.org/)
+- Code Reviewers
+- Coding Partners
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] It would be better if we fix the MIT and Issues page link, these links are broken and needs to be fixed :man_technologist:

Reply:
@addod19 Thank you for bringing this to my attention. I appreciate your feedback and have taken note of your suggestion to fix the broken links to the MIT License and Issues page. I apologize for any inconvenience this may have caused and will work to resolve the issue as soon as possible. Thank you again for taking the time to let me know.